x/gov/client/testutil/query.go

```diff
@@ -24,23 +24,32 @@ func (s *IntegrationTestSuite) TestCmdParams() {
 		{
 			"json output",
 			[]string{fmt.Sprintf("--%s=json", tmcli.OutputFlag)},
-			`{"voting_params":{"voting_period":"172800000000000"},"tally_params":{"quorum":"0.334000000000000000","threshold":"0.500000000000000000","veto_threshold":"0.334000000000000000"},"deposit_params":{"min_deposit":[{"denom":"stake","amount":"10000000"}],"max_deposit_period":"172800000000000"}}`,
+			`{"voting_params":{"voting_period":"172800s"},"deposit_params":{"min_deposit":[{"denom":"stake","amount":"10000000"}],"max_deposit_period":"172800s"},"tally_params":{"quorum":"0.334000000000000000","threshold":"0.500000000000000000","veto_threshold":"0.334000000000000000"},"params":{"min_deposit":[{"denom":"stake","amount":"10000000"}],"max_deposit_period":"172800s","voting_period":"172800s","quorum":"0.334000000000000000","threshold":"0.500000000000000000","veto_threshold":"0.334000000000000000"}}`,
 		},
 		{
 			"text output",
 			[]string{},
 			`
 deposit_params:
-  max_deposit_period: "172800000000000"
+  max_deposit_period: 172800s
```

Comment:
Do you know why this output changed? The new one looks better, but just making sure

Reply:
may be because this change: https://github.com/cosmos/cosmos-sdk/pull/12631/files#diff-28aea802796de8ea1a531922b1cfdf0ee2bdab880f6289592c11de066bbabd22R550